OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListFragment.kt

```diff
@@ -124,23 +127,20 @@ class MyCoursesListFragment : OfflineSupportBaseFragment(), RefreshListener {
             binding.loadingIndicator.root.visibility = View.VISIBLE
             errorNotification.hideError()
         }
-        if (isCached) {
-            getUserEnrolledCourses(courseAPI.enrolledCoursesFromCache, isCached = true)
-        } else {
-            getUserEnrolledCourses(courseAPI.enrolledCourses)
-        }
+        enrolledCoursesApi = if (isCached) courseAPI.enrolledCoursesFromCache else courseAPI.enrolledCourses
```

Comment:
isCached should be renamed to fromCache and javadoc should be updated accordingly.

Reply:
Renamed